l1-contracts/test/foundry/unit/concrete/TransactionValidator/_TransactionValidator_Shared.t.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {TransactionValidatorTest} from "solpp/dev-contracts/test/TransactionValidatorTest.sol";
+import {IMailbox} from "solpp/zksync/interfaces/IMailbox.sol";
+
+contract TransactionValidatorSharedTest is Test {
+    TransactionValidatorTest internal validator;
```

Comment:
`TransactionValidatorTest` only contains two functions wrapper over `TransactionValidator` library. Maybe remove `TransactionValidatorTest` at all and call internal functions from `TransactionValidator`? This will remove legacy test contract and make coverage cleaner.